source/fundamentals/crud/write-operations/bulk-write.txt

```diff
@@ -0,0 +1,550 @@
+       |
+       | **Data Type:** {+string-data-type+} or `CollectionNamespace <{+new-api-root+}/MongoDB.Driver/MongoDB.Driver.CollectionNamespace.html>`__
+
+   * - ``filter``
+     - | The **query filter** that specifies the criteria used to match documents in your collection.
+         The ``UpdateOne`` operation updates *only the first document* that matches the
+         query filter.
+       |
+       | **Data Type:** `FilterDefinition<TDocument> <{+new-api-root+}/MongoDB.Driver/MongoDB.Driver.FilterDefinition-1.html>`__
+
+   * - ``update``
+     - | The update operation you want to perform. For more information about update
+         operations, see :manual:`Field Update Operators </reference/operator/update-field/>`
+         in the {+mdb-server+} manual.
+       |
+       | **Data Type:** `UpdateDefinition<TDocument> <{+new-api-root+}/MongoDB.Driver/MongoDB.Driver.UpdateDefinition-1.html>`__
+
+   * - ``collation``
+     - | *Optional.* The language collation to use when sorting results. See the
+         :manual:`{+mdb+server+} manual</reference/command/delete/#std-label-deletes-array-collation>`
+         for more information.
+       |
+       | **Data Type:** `Collation <{+new-api-root+}/MongoDB.Driver/MongoDB.Driver.Collation.html>`__
+       | **Default:** ``null``
+   
+   * - ``hint``
+     - | *Optional.* The index to use to scan for documents. 
+         See the :manual:`{+mdb-server+} manual</reference/command/update/#std-label-update-command-hint>`
+         for more information.
+       |
+       | **Data Type:** `BsonValue <{+new-api-root+}/MongoDB.Bson/MongoDB.Bson.BsonValue.html>`__
+       | **Default:** ``null``
+
+   * - ``isUpsert``
+     - | *Optional.* Specifies whether the update operation performs an upsert operation if no 
+         documents match the query filter. See the :manual:`{+mdb-server+} manual</reference/command/update/#std-label-update-command-upsert>`
+         for more information.
+       |
+       | **Data Type:** {+bool-data-type+}
+       | **Default:** ``false``
+
+   * - ``arrayFilters``
+     - | Specifies which array elements to modify for an update operation on an array field.
+         See the :manual:`{+mdb-server+} manual</reference/command/update/#update-elements-match-arrayfilters-criteria>`
+         for more information.
+       |
+       | **Data Type:** `IEnumerable<ArrayFilterDefinition> <https://learn.microsoft.com/en-us/dotnet/api/system.collections.generic.ienumerable-1?view=net-8.0>`__ 
+       | **Default:** ``null``
+
+In the following code example, the ``BulkWriteUpdateOneModel<BsonDocument>`` object
+represents an update operation on the ``sample_restaurants.restaurants`` collection.
+The operation matches the first document in the collection where the value of the ``name``
+field is ``"Mongo's Deli"``. It then updates the value of the ``cuisine`` field in the
+matched document to ``"Sandwiches and Salads"``.
```

Comment:
This sounds good to me as a model for the other sections!

Reply:
Think i got them all